### PR TITLE
Jupyter notebook cherry pick

### PR DESCRIPTION
**Pull request by chiayi**
Cherry-pick notebook test to release-1.1

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -365,11 +373,11 @@ steps:
           exit 1
         fi
 
-        # TODO: re-add this check once generating embeddings in CI is stable
-        # if [[ $(cat /workspace/rag_prompt_result.txt) != "pass" ]]; then
-        #   echo "rag frontend test failed"
-        #   exit 1
-        # fi
+        TODO: re-add this check once generating embeddings in CI is stable
```
Comment (andrewsykim) [inline cloudbuild.yaml:376]: remove this TODO since it's no longer a comment

Comment (chiayi) [reply cloudbuild.yaml]: Added the PR with this change.

File: cloudbuild.yaml [hunk 2]
```diff
@@ -365,11 +373,10 @@ steps:
           exit 1
         fi
 
-        # TODO: re-add this check once generating embeddings in CI is stable
-        # if [[ $(cat /workspace/rag_prompt_result.txt) != "pass" ]]; then
-        #   echo "rag frontend test failed"
-        #   exit 1
-        # fi
+        if [[ $(cat /workspace/rag_prompt_result.txt) != "pass" ]]; then
+          echo "rag frontend test failed"
```
Comment (andrewsykim) [inline cloudbuild.yaml:377]: fix in a future PR: this should be:
```
echo "rag prompt test failed"
```

**Review by andrewsykim — APPROVED** [review]

File: cloudbuild.yaml (hunk 2 above)
Comment (chiayi) [reply cloudbuild.yaml:377]: Okay, will open a future PR for this fix.

**Comment by andrewsykim** [comment]
@chiayi the presubmit is going to fail due to the HF issue -- I've marked presubmit as optional for now

**Comment by chiayi** [comment]
> @chiayi the presubmit is going to fail due to the HF issue -- I've marked presubmit as optional for now

Was just running to make sure that the changes didn't effect anything and other parts of the cloud build still ran

**Pull request merged**